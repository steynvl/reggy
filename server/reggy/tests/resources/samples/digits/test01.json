{
  "sampleStringsInfo": [
    [
      {
        "markerType": "Digits",
        "markedStrings": [
          "adsad"
        ],
        "markerInfo": {
          "zero": true,
          "one": true,
          "two": true,
          "three": true,
          "four": true,
          "five": true,
          "six": true,
          "seven": true,
          "eight": true,
          "nine": true,
          "minus": {
            "minus": false,
            "optional": false
          }
        },
        "repeatInfo": {
          "repeat": "1"
        }
      }
    ]
  ],
  "generalRegexInfo": {
    "startRegexMatchAt": "Anywhere",
    "endRegexMatchAt": "Anywhere",
    "regexTarget": "Java"
  },
  "generateMethod": "samplesAndSemantics"
}
