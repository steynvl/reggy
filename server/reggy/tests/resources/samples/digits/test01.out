Java
Pattern regex = Pattern.compile("\\d");
\\d
Perl
my $regex = /\d/;
\d
POSIX
\d
\d
Python
regex = re.compile(r'\d')
\d
JavaScript
const regex = /\d/;
\d
PHP
$regex = '/\d/';
\d
Golang
regex, _ := regexp.Compile("\\d")
\\d
Rust
let re = Regex::new(r"\d").unwrap();
\d
C#
var re = new Regex(@"\d");
\d
Scala
val re = "\\d".r
\\d
Kotlin
val regex = Regex("\\d")
\\d